en/index.md

```diff
@@ -524,6 +524,42 @@ For example:
 
 - **Don’t use:** Click the ![pencil](https://assets.ubuntu.com/v1/633aab2b-pencil.png) pencil icon to edit the post.
 
+## Hyperlinks
+
+Here are some pointers about the general use of hyperlinks and how to format them correctly.
+
+### General use
+
+Avoid excessive links in the same paragraph or instruction. If you find yourself introducing several links in your content, consider listing them in a separate section called "Related topics", "Additional resources", or similar.
```

Comment:
I know there are other parts which ignore this and I know that discourse does things differently, but can we word-wrap to 79? It makes reviewing so much easier